src/screens/Ticketing/Ticket/TicketInfo.tsx

```diff
@@ -249,38 +361,42 @@ const TicketInspectionSymbol = ({
   const showAsInspectable = isInspectable || status !== 'valid';
   const isValid = status === 'valid';
   return (
-    <View
-      style={[
-        showAsInspectable && styles.symbolContainer,
-        isValid && {
-          ...styles.symbolContainerCircle,
-          backgroundColor: themeColor
-            ? flatStaticColors[themeName][themeColor].background
-            : undefined,
-        },
-        isValid &&
-          !isInspectable && {
-            ...styles.textContainer,
-            borderColor: theme.static.status.warning.background,
+    <View style={styles.symbolContainer}>
+      <View
+        style={[
+          showAsInspectable && styles.symbolIcon,
+          isValid && {
```

Comment:
I know this isn't a change from before but national park rules and all, this looks like something that really could use some naming and abstractions. Hard to read the logic here.